code/modules/overmap/disperser/disperser_console.dm

```diff
@@ -150,7 +150,7 @@
 		data["range"] = range
 		data["next_shot"] = round(get_next_shot_seconds())
 		data["nopower"] = !data["faillink"] && (!front.powered() || !middle.powered() || !back.powered())
-		data["skill"] = user.get_skill_value(core_skill) > skill_offset
+		data["skill"] = TRUE // todo: skills
```

Comment:
todo (:

Reply:
yeah this kind of just tells the interface that the user is skilled enough & disperser is now hardcoded to work (as opposed to the old thing which hardcoded the skills to always be at max, which works too but was kind of just as silly)